src/vite.config.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C)  Online-Go.com
+ * Copyright (C)  Ben Jones
```

Comment:
Probably don't need to assign copyright to Ben Jones, I reckon you copied this from something that he was doing that he felt he wanted to be able to safely copy out and assign his own license to and all of that.. but that's not you so can probably just remove that from this  :)